flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/table/RowDataToAttributeValueConverterTest.java

```diff
@@ -51,7 +58,7 @@ void testChar() {
 
         DataType dataType = DataTypes.ROW(DataTypes.FIELD(key, DataTypes.CHAR(9)));
         RowDataToAttributeValueConverter rowDataToAttributeValueConverter =
-                new RowDataToAttributeValueConverter(dataType);
+                new RowDataToAttributeValueConverter(dataType, null);
```

Comment:
Here and below, you can remove `null` argument since you've added constructor overload

Reply:
Thanks!  Implemented that change.